docs/book/introduction.md

```diff
@@ -4,27 +4,204 @@ description: Welcome to ZenML!
+development, staging, and production infrastructures as 
+[Stacks](./getting-started/core-concepts.md#2-execution), you can move 
+entire ML workflows to different environments in seconds.
+
+```bash
+zenml stack set staging
+python run.py  # test your workflows on staging infrastructure
+zenml stack set production
+python run.py  # run your workflows in production
+```
+
+**Reproducibility:** ZenML enables you to painlessly reproduce previous results
+by automatically tracking and versioning all stacks, pipelines, artifacts, and
+source code. In the ZenML dashboard, you can get an overview of everything that
+has happened and drill down into detailed lineage visualizations.
+Try it out at [https://demo.zenml.io/](https://demo.zenml.io/)!
+
+<figure>
+<img src=".gitbook/assets/intro_dashboard.png" alt="ZenML Dashboard Overview" width="40%">
+</figure>
+
+**Automated Deployments:** With ZenML, you no longer need to upload custom 
+Docker images to the cloud whenever you want to deploy a new model to 
+production. Simply define your ML workflow as a ZenML pipeline, let ZenML 
+handle the containerization, and have your model automatically deployed to 
+a highly scalable Kubernetes deployment service like 
+[Seldon](./user-guide/component-galery/model-deployers/seldon.md).
+
+```python
+from zenml.integrations.seldon.steps import seldon_model_deployer_step
+
+from my_organization.steps import data_loader_step, model_trainer_step
+
+
+@pipeline
+def my_pipeline():
+  data = data_loader_step()
+  model = model_trainer_step(data)
+  seldon_model_deployer_step(model)
+```
+
+{% endtab %}
+
+{% tab title="ZenML for MLOps / Platform Engineers" %}
+
+ZenML enables MLOps infrastructure experts to define, deploy, and manage
+sophisticated production environments that are easy to share with colleagues.
+
+**Built-in Deployment:** ZenML can be deployed on any cloud provider and 
+provides many Terraform-based utility functions to deploy other MLOps tools or 
+even entire MLOps stacks:
+
+```bash
+zenml deploy --provider aws  # Deploy ZenML to any cloud
+zenml orchestrator deploy kfp --flavor kubeflow --cloud gcp  # Deploy MLOps tools and infrastructure to any cloud
+zenml stack recipe deploy gcp-vertexai  # Deploy entire MLOps stacks at once
+```
+
+**Standardization:** With ZenML, you can standardize MLOps infrastructure and 
+tooling across your organization. Simply register your staging and production 
+environments as ZenML stacks and invite your colleagues to run ML workflows on 
+them.
+
+```bash
+zenml orchestrator register kfp_orchestrator -f kubeflow  # Register MLOps tools and infrastructure
+zenml stack register production --orchestrator kubeflow ...  # Register your production environment
+zenml stack share production  # Make it available to your colleagues
+```
+
+Registering your environments as ZenML stacks also enables you to browse and
+explore all your infrastructure setups in a convenient user interface. 
```

Comment:
'infrastructure setups'? if you're saying 'stacks' earlier in the sentence, why do you have to use this new phrasing?